uml/backend-software-system/screen-handler.puml

```diff
@@ -0,0 +1,51 @@
+@startuml
+
+package "Screen Handler" {
+  class EventType {
+    type : String
+    sprite : Sprite
+  }
+
+  class SpritesReader {
+    +readSprites() : List<Sprite>
+  }
+
+  class ScreenManager {
+    -sprites : List<Sprite>
+    -mapper : ScreenMapper
+    -events : ScreenEvents
+    -settings : ScreenSettings
+    +showSprites(sprites : List<Sprite>)
+  }
+
+  class ScreenMapper {
+    +showSprites(sprites : List<Sprite>)
+    +updateSprite(sprite : Sprite)
+  }
+
+  class ScreenEvents {
+    +listenEvents() : void
+    +onEvent(event : EventType) : void
+  }
+
+  class ScreenSettings {
+    soundVolume : Double
+    musicVolume : Double
+    language : String
+    +getSoundVolume() : Double
+    +setSoundVolume(volume : Double) : void
+    +getMusicVolume() : Double
+    +setMusicVolume(volume : Double) : void
+    +getLanguage() : String
+    +setLanguage(language : String) : void
```

Comment:
The sound logic will be handled by "Sound Manager" please erase this.

Reply:
I think Sound Manager package only refers to reproduction and its default settings of the audio provided by the game developer but the main idea of this class was to show user a screen with those information. I mean something like this.

![image](https://github.com/Pending-Name-21/arquitecture/assets/73414963/dff16e0a-1e3f-4d58-b0fe-5903b52f5b7a)

As you can see there, user can increase or decrease the level of audio and also it shows the users some extra information like the date, time and battery percentage.

Reply:
That's a great idea. But it's out of scope, our profile says there won't be a UI for the console. But only the game itself

Reply:
Okay!